include/lsst/afw/table/Schema.h

```diff
@@ -171,7 +184,7 @@ class Schema {
      *  @brief Add a new field to the Schema, and return the associated Key.
      *
      *  This is simply a convenience wrapper, equivalent to:
-     *  @code
+     *  @codeinspect
```

Comment:
What is the `codeinspect` command? It is not listed in https://www.stack.nl/~dimitri/doxygen/manual/commands.html and a google search didn't turn up anything.